features/support/page_objects/summary_page_error.rb

```diff
@@ -0,0 +1,33 @@
+class SummaryPageError < BasePage
```

Comment:
As I mentioned above I don't see reason for this file at all. You can just add the addition to summary_page object.